### PR TITLE
Catching up to ember-simple-auth jj-abrams branch

### PR DESCRIPTION
**Pull request by mmahalwy**
I'll fix the tests soon - just need to make time for it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-simple-auth-token",
-  "version": "0.7.3",
+  "version": "0.7.4",
```
Comment (kmiyashiro) [inline package.json]: I don't think you're supposed to change the version of a lib in a PR

Comment (mmahalwy) [inline package.json]: Good call! I will revert that

**Comment by jpadilla** [comment]
@mmahalwy thanks for working on this. Should I jump in to get it finished now that 1.0 was released or will you be able to wrap this up?

**Comment by mmahalwy** [comment]
@jpadilla that'd be great! I have my hands caught up right now :(

**Comment by wayne-o** [comment]
If I have time to look into this today I will try and move it forward.

**Comment by acorncom** [comment]
Guys, I just opened another pull request with some more work done on this.  Should we create an update branch here on the main repo that we can all work against instead of working from various forks?

Looks like we're down to one set of failing tests (all the `token-test.js` tests).  I'm running out of time today, but might be able to revisit this next week.  Would love to have someone else help push this over the finish line before I get back though :smile:

**Comment by mmahalwy** [comment]
@acorncom thanks! I will try to look at this soon

**Comment by jpadilla** [comment]
Thanks. #100 was merged and I'll be releasing an update during the holidays. Feel free to check out master.

**Pull request closed without merging**